mmdet/models/losses/cross_entropy_loss.py

```diff
@@ -104,7 +104,7 @@ def binary_cross_entropy(pred,
     if weight is not None:
         weight = weight.float()
     loss = F.binary_cross_entropy_with_logits(
-        pred, label.float(), pos_weight=class_weight, reduction='none')
+        pred, label.float(), weight=class_weight, reduction='none')
```

Comment:
Please take a look at the docstring of `F.binary_cross_entropy_with_logits`.
`weight` should be a tensor that matches the input tensor shape. Not the class-aware weight.
`pos_weight` should be a vector with a length equal to the number of classes.

Reply:
it will broadcasts to (....., C)
Of course, the premise is to convert pred tensor into shape of (..., C). the yolov3 in mmdet has done this transform

Reply:
pos_weight is a vector with a length equal to the number of classes， but it is different with the classwise weight. Using pos_weight will pos items and neg items loss unblanced.